.github/workflows/main.yml

```diff
@@ -33,25 +73,41 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
-    - name: Upgrade packaging tools
-      run: python -m pip install --upgrade pip setuptools virtualenv wheel
-
     - name: Install dependencies
-      run: python -m pip install --upgrade codecov tox
+      run: |
+        python -m pip install --upgrade pip setuptools virtualenv wheel
+        python -m pip install --upgrade codecov tox tox-gh-actions
 
-    - name: Run tox targets for ${{ matrix.python-version }}
+    - name: Run tox
+      if: ${{ matrix.tox-env == 'N/A' }}
       run: |
-        ENV_PREFIX=$(tr -C -d "0-9" <<< "${{ matrix.python-version }}")
-        TOXENV=$(tox --listenvs | grep "^py$ENV_PREFIX" | tr '\n' ',') tox
+        tox
+      env:
+        DJANGO: ${{ matrix.django-version }}
 
+    # This would not be needed after https://github.com/ymyzk/tox-gh-actions/pull/60
+    # but it may need adjustment for --installpkg
     - name: Run extra tox targets
-      if: ${{ matrix.python-version == '3.9' }}
+      if: ${{ matrix.tox-env != 'N/A' }}
       run: |
-        python setup.py bdist_wheel
-        rm -r djangorestframework.egg-info  # see #6139
-        tox -e base,dist,docs
-        tox -e dist --installpkg ./dist/djangorestframework-*.whl
+        ARGS=()
+        if [[ $TOXENV == 'dist' ]]; then
```

Comment:
I don't think this is still needed. I'm not seeing any issues running this on my local machine